bzt/modules/aggregator.py

```diff
@@ -43,13 +43,27 @@ def __init__(self, low, high, sign_figures):
         self._cached_perc = None
         self._cached_stdev = None
 
+    def __deepcopy__(self, memo):
+        new = RespTimesCounter(self.low, self.high, self.sign_figures)
+        new._cached_perc = self._cached_perc
+        new._cached_stdev = self._cached_stdev
+
+        # TODO: maybe hdrpy can encapsulate this itself
```

Comment:
maybe hdrpy can encapsulate this itself

Reply:
i agree, hdrpy totally should do it itself